samples/main.cpp

```diff
@@ -71,7 +78,7 @@ int main(int argc, char *argv[])
   auto vec = playButtonTransform.getPosition();
   vec.setX(playButtonTransform.getPosition().getX() - 75);
   playAudioTextTransform.setPosition(vec);
-  playAudioText = runner.getRenderer()->createTextRect(playAudioTextTransform, 1, NovelRT::Graphics::RGBAConfig(0, 0, 0, 255), 36, "Gayathri-Regular.ttf");
+  playAudioText = runner.getRenderer()->createTextRect(playAudioTextTransform, 1, NovelRT::Graphics::RGBAConfig(0, 0, 0, 255), 36, (fontsDirPath / "Gayathri-Regular.ttf").string());
   playAudioText->setText("Play Audio");
```

Comment:
Not visible here, but further down the audio service only takes in a file name (such as `sparta.wav`): https://github.com/novelrt/NovelRT/pull/107/files#diff-ef810af08ed542e7c7cf6edddb5bd627R113)

Ideally, we would have the asset manage track the full paths and we'd be able to internally track these via custom ids (ints, strings, or something else). That way, you can have `<custom-path>/sparta.wav` as the loaded asset, but still access it via `sparta.wav`